src/effects/nyquist/Nyquist.cpp

```diff
@@ -943,15 +943,15 @@ bool NyquistEffect::Process(EffectInstance &, EffectSettings &settings)
          if (bOnePassTool) {
          }
          else {
-            if (auto channels = TrackList::Channels(mCurChannelGroup)
+            if (auto channels = mCurChannelGroup->Channels()
                ; channels.size() > 1
             ) {
                // TODO: more-than-two-channels
                // Pay attention to consistency of mNumSelectedChannels
                // with the running tally made by this loop!
                mCurNumChannels = 2;
 
-               mCurTrack[1] = * ++ channels.first;
+               mCurTrack[1] = &** ++ channels.first;
```

Comment:
Completely unreadable. Please at least use parenthesis to clarify the meaning. 

Original code was bad, but this is just esoteric.

Reply:
I'll make it `(* ++ channels.first).get()` and similar wherever I wrote `&**`